src/v/redpanda/admin/partition.cc

```diff
@@ -16,11 +16,15 @@
 #include "cluster/partition_manager.h"
 #include "cluster/shard_table.h"
 #include "cluster/topics_frontend.h"
+#include "container/fragmented_vector.h"
 #include "redpanda/admin/api-doc/debug.json.hh"
 #include "redpanda/admin/api-doc/partition.json.hh"
 #include "redpanda/admin/server.h"
 #include "redpanda/admin/util.h"
 
+#include <seastar/coroutine/maybe_yield.hh>
```

Comment:
question: Was this include intentionally added?  I ask because I don't see a yield in this commit